src/sentry/static/sentry/app/views/settings/components/forms/rangeField.jsx

```diff
@@ -0,0 +1,148 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+const Slider = styled.input`
+  -webkit-appearance: none;
+  width: 100%;
+  margin: ${p => p.theme.grid}px 0;
+
+  &:focus {
+    outline: none;
+    &::-webkit-slider-runnable-track {
+      background: ${p => p.theme.borderDark};
+    }
+  }
+
+  &::-webkit-slider-runnable-track {
+    width: 100%;
+    height: 3px;
+    cursor: pointer;
+    background: ${p => p.theme.borderLight};
+    border-radius: 25px;
+    border: 0;
+  }
+
+  &::-webkit-slider-thumb {
+    box-shadow: 0 0 0 3px #fff;
+    height: 17px;
+    width: 17px;
+    border-radius: 50%;
+    background: ${p => p.theme.purple};
+    cursor: pointer;
+    -webkit-appearance: none;
+    margin-top: -7px;
+    border: 0;
+  }
+
+  &::-moz-range-track {
+    width: 100%;
+    height: 3px;
+    cursor: pointer;
+    background: ${p => p.theme.borderLight};
+    border-radius: 25px;
+    border: 0;
+  }
```

Comment:
gettin a little lazy on the returns here and prettier isn't gonna format it for us I don't think!

Reply:
Can you provide a little more context for this piece of feedback?

Reply:
oh sorry! there is a single return between css blocks in most of our code, but there isn't any here due to the copy-paste stuff I think. Normally a linter would handle this, but we can't lint this I think.

Reply:
but it looks like you got it so nvm

Reply:
IGNORE ME

Reply:
Oh, got it. I actually missed a couple instances at the beginning of the file. Fixed!